std/digest/digest.d

```diff
@@ -579,6 +579,15 @@ interface Digest
         @trusted nothrow void put(scope const(ubyte)[] data...);
 
         /**
+         * This is a conveniance function to pass some strings with char
+         * as element encoding type.
+         */
+        @trusted nothrow final void putText(scope const(char)[] data...)
```

Comment:
The reason why this is not an overload is that if it's named "put" then it's not selected during one of the test. If it's not preimplemented in the interface and implemented in the Wrapper then another test fail (one where two ubytes are passed).

This's been a problem when the PR was sent because I wasn't recompiling phobos with the unittest flag and once it was online, the problem appeared and this was the only quick fix I've found.

Short demo of the problem:

```D
interface I
{
    void test(const(ubyte)[] p);
    final void test(const(char)[] p){test(cast(const(ubyte)[])p);}
    final void othername(const(char)[] p){test(cast(const(ubyte)[])p);}
}

class C: I
{
    void test(const(ubyte)[]){}
}


void main()
{
    (new C).test("a"); // ouch
    //(new C).othername("a"); // OK, like putText
    //(cast(I) (new C)).test("a"); // OK
} 
```